cloud-gateway-service/build.gradle

```diff
@@ -71,12 +71,15 @@ dependencies {
         exclude group: "org.springframework", module: "spring-core"
         exclude group: "org.springframework", module: "spring-web"
     }
+    // https://mvnrepository.com/artifact/org.springframework.boot/spring-boot-starter-security
+    implementation 'org.springframework.boot:spring-boot-starter-security:2.7.7'
```

Comment:
could you define it in the `version.gradle`?

Reply:
there is a different version of spring-boot-starter-security defined in version.gradle, it would be better to update the springbootversion to the latest minor but we should do it in a separate PR as this would affect almost every component